public-api/src/main/java/org/pmiops/workbench/publicapi/DataBrowserController.java

```
@@ -557,26 +567,41 @@ public SurveyQuestionResult apply(AchillesResult o) {
             if (criteriaList.size() >= 1) {
                 criteriaList.remove(parent);
             }
+            if (parent == null) {
+                throw new DataNotFoundException("Cannot find rolled up counts of this concept");
+            }
             response.setParent(TO_CLIENT_CBCRITERIA.apply(parent));
             Multimap<Long, CBCriteria> parentCriteria = Multimaps
                     .index(criteriaList, CBCriteria::getParentId);
             return ResponseEntity.ok(response);
+        } else {
+            throw new DataNotFoundException("Cannot find rolled up counts of this concept");
```

Why throw an exception instead of returning an empty response?

---

Done. Thanks.